src/components/Weather/Weather.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { WeatherBody } from "./WeatherBody";
+import "./WeatherBody.css";
+
+const Weather = () => {
+  const [lat, setLat] = useState([]);
+  const [long, setLong] = useState([]);
+  const [data, setData] = useState([]);
+
+  const REACT_APP_API_URL = "https://api.openweathermap.org/data/2.5";
+  const REACT_APP_API_KEY = "c962ed695546773fa54252b994d33923";
```

Comment:
Can use the env file for storing these credentials at a safe point by passing the .env in gitignore file and passing it to the deploy platform as well in order to make sure the app works on further previews!!